### PR TITLE
mainboard/protectli/vault_jsl/mainboard.c: Disable AER for WIFI slot

### PR DESCRIPTION
**Pull request by miczyg1**

### CODE REVIEW

**Comment by macpijan** [comment]
@krystian-hebel is experiencing this, so I will let him confirm.

File: src/mainboard/protectli/vault_jsl/mainboard.c [hunk 1]
```diff
@@ -20,7 +20,7 @@ void mainboard_silicon_init_params(FSP_S_CONFIG *params)
 	params->PcieRpAdvancedErrorReporting[0] = 1;
 	params->PcieRpAdvancedErrorReporting[1] = 1;
 	params->PcieRpAdvancedErrorReporting[3] = 1;
-	params->PcieRpAdvancedErrorReporting[4] = 1;
+	params->PcieRpAdvancedErrorReporting[4] = 0; // Atheros WiFi cards don't like AER
```
Comment (krystian-hebel) [inline src/mainboard/protectli/vault_jsl/mainboard.c:23]: Please try disabling ASPM L0s instead of hiding valid errors.

**Comment by krystian-hebel** [comment]
Proper fix is in https://github.com/Dasharo/coreboot/pull/535, after testing on V1610 this PR probably can be closed.

**Comment by miczyg1** [comment]
Supersede by https://github.com/Dasharo/coreboot/pull/535

**Pull request closed without merging**